### DISCUSSION TITLE
Shallow Documentation: Adding Images, Fonts, and Files

### DISCUSSION THREAD

**Discussion by kidalikev** [nrwl/nx#6936] [General]
**Adding Images, Fonts, and Files**
Your documentation on the above topic is extremely shallow or the title is misleading, every small details matters.
There is no where it demonstrates how to import custom fonts as the title suggests. A title should reflect the content.

ref: https://nx.dev/latest/react/guides/adding-assets-react

**Comment by Vadi** [comment]
Agree. Currently stumbling on the same thing. Where to place TTF files? Keeping it in `asset` folder does not show up in `dist` folder.

**Comment by hokecyber** [comment]
Very disappointing and unprofessional for such good tool

**Comment by arturohernandez10** [comment]
Unprofessional, is a bit much from something free. How can we help? I'll write it if I'm pointed in the right direction. 

Anyone interested, please comment here what information is missing, Understandably several suggestions may be misaligned, because of the same issue. But please include a couple of sentences with detail of what could be in the docs.

**Comment by hokecyber** [comment]
@arturohernandez10 I apologize for my foul language and I take it back. I was carried away by the fact this has taken maybe years ( I reported this on Sep 4, 2021) or many months without any closure yet in different forums I see the community yawning to have answers,
The documentation for this super good tool is missing something (Ref - https://nx.dev/latest/react/guides/adding-assets-react), the title says Adding fonts yet nowhere is it written how

**Comment by AnasPlusAnas** [comment]
Well, anyone figured out how to? The doc is still not updated.
